libs/src/services/creatorNode/CreatorNodeSelection.ts

```diff
@@ -208,22 +317,30 @@ class CreatorNodeSelection extends ServiceSelection {
   /**
    * Performs a sync check for every endpoint in services. Returns an array of successful sync checked endpoints and
    * adds the err'd sync checked endpoints to this.unhealthy
-   * @param {string[]} services content node endpoints
-   * @param {number?} timeout ms applied to each request
+   * @param services content node endpoints
+   * @param timeout ms applied to each request
    */
-  async _performSyncChecks (services, timeout = null) {
-    const successfulSyncCheckServices = []
-    const syncResponses = await Promise.all(services.map(service => this.getSyncStatus(service, timeout)))
+  async _performSyncChecks(services: ServiceName[], timeout: Timeout = null) {
+    const successfulSyncCheckServices: ServiceName[] = []
+    const syncResponses = await Promise.all(
+      services.map(
+        async (service) => await this.getSyncStatus(service, timeout)
+      )
+    )
     // Perform sync checks on all services
     for (const response of syncResponses) {
       // Could not perform a sync check. Add to unhealthy
       if (response.error) {
-        console.warn(`CreatorNodeSelection - Failed sync status check for ${response.service}: ${response.error}`)
+        console.warn(
+          `CreatorNodeSelection - Failed sync status check for ${response.service}: ${response.error}`
+        )
         this.addUnhealthy(response.service)
```

Comment:
Not sure what the proper behavior is here actually – maybe this hsould be considered unhealthy?

Reply:
yeah me neither, ultimately just a lint fix here? want me to add a TODO for someone familiar with this code?